server/index.js

```diff
@@ -52,11 +53,11 @@ const startListening = async () => {
   app.listen(PORT, () => console.log(`Mixing it up on port ${PORT}`))
 }
 
-const autoCleanDB = async () => {
-  await Pin.findandRemoveOldPins()
-  setInterval(async () => {
-    await Pin.findandRemoveOldPins()
-  }, 1209600000)
+const autoCleanDB = async (ttl, interval = 1209600000) => {
+  await Pin.findandRemoveOldPins(ttl)
```

Comment:
small thing - but now that we've changed this autoCleanDB to use the scheduler, we no longer need this initial call. That happens in the Scheduler.